packages/workflow/test/AugmentObject.test.ts

```diff
@@ -290,7 +290,7 @@ describe('AugmentObject', () => {
 		});
 
 		test('should work with complex values on first level', () => {
-			const originalObject = {
+			const originalObject: any = {
```

Comment:
I take it there is no other way?

Reply:
in this case, unfortunately not. it's not worth defining a complex type here just for the test.